openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreQuickFilters.tsx

```diff
@@ -43,19 +47,34 @@ const ExploreQuickFilters: FC<ExploreQuickFiltersProps> = ({
   const [options, setOptions] = useState<SearchDropdownOption[]>();
   const [isOptionsLoading, setIsOptionsLoading] = useState<boolean>(false);
 
+  const fetchDefaultOptions = async (
+    index: SearchIndex | SearchIndex[],
+    key: string
+  ) => {
+    const res = await getAdvancedFieldDefaultOptions(index, key);
+
+    const buckets = res.data.aggregations[`sterms#${key}`].buckets;
+
+    const optionsArray = buckets.map((option) => ({
+      key: option.key,
+      label: option.key,
+    }));
+
+    setOptions(uniqWith(optionsArray, isEqual));
+  };
+
   const getInitialOptions = async (key: string) => {
     setIsOptionsLoading(true);
     setOptions([]);
     try {
-      const res = await getAdvancedFieldDefaultOptions(index, key);
-      const buckets = res.data.aggregations[`sterms#${key}`].buckets;
-
-      const optionsArray = buckets.map((option) => ({
-        key: option.key,
-        label: option.label ?? option.key,
-      }));
-
-      setOptions(uniqWith(optionsArray, isEqual));
+      if (key === MISC_FIELDS[0]) {
+        await fetchDefaultOptions(
+          [SearchIndex.USER, SearchIndex.TEAM],
+          OWNER_QUICK_FILTER_DEFAULT_OPTIONS_KEY
+        );
```

Comment:
Why aren't we using response?

Reply:
The function `fetchDefaultOptions` is in the component itself, we use response in that